code/modules/power/cable.dm

```diff
@@ -415,6 +415,11 @@
 
 	return
 
+#define CHECK_AHEAD_LIST list(NORTH, EAST, NORTHWEST, NORTHEAST)
```

Comment:
this is literally only used once, why is it a define

Reply:
ah, lol
i thought it'd be used more but i was wrong